README.md

```diff
@@ -78,6 +92,37 @@ For information about contributing to this project please refer to this
 * Releasing new versions
 * Docker development
 
+## CI/Testing Strategy
+
+The following parameters are considered in the testing strategy
+in the CI for this project:
+### Database
+
+We are testing this starter kit against PostgreSQL and MySQL.
+### Ruby Version
+
+We are testing this starter kit against the latest Ruby version
+and the last minor of the previous major. Eg: 3.2 and 2.7.
+
+### Rails version
+
+We are testing this starter kit against the last Rails version
+at the time the corresponding Solidus version has been released.
```

Comment:
Once Rails v7.1 is out and we're still on v3.3, that would translate into not covering the latest Rails, which will also be the most common use case. What do you think about testing against the latest Rails supported by any Solidus version?

Reply:
But Solidus `v3.3` won't be supporting Rails `v7.1` and we would need to wait for Solidus `v3.4` to be released. That why I said `the last Rails version at the time the corresponding Solidus version has been released`. Does it sound correct? Or are you proposing to change the supported version exactly when Rails releases new versions?

Reply:
You're right, I was confused.

Reply:
Still, we need to be more clear about the Rails versions supported in the installation guides: when Rails 7.1 is out, it's legit to expect people to use it with a fresh Solidus installation.

Reply:
But... When Rails 7.1 is out, Solidus `v3.3` [won't be compatible with that version of Rails](https://github.com/solidusio/solidus/blob/v3.3/core/solidus_core.gemspec#L30). So formally, in terms of dependencies, we are safe.